apps/src/code-studio/components/Libraries/LibraryCreationDialog.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BaseDialog from '../../../templates/BaseDialog';
+import {connect} from 'react-redux';
+import {hideLibraryCreationDialog} from './libraryCreationRedux';
+import libraryParser from './libraryParser';
+import LibraryClientApi from './LibraryClientApi';
+import i18n from '@cdo/locale';
+
+class LibraryCreationDialog extends React.Component {
```

Comment:
No tests added for this yet because no part of this is in its final state. It is only here to give some scaffolding that connects the redux portions to the publishing api.